oneview_redfish_toolkit/util.py

```diff
@@ -436,3 +441,43 @@ def dispatch_event(event):
             globals()['delivery_retry_interval'])
 
         dispatcher.start()
+
+
+def check_oneview_availability(ov_config):
+    """Check OneView availability by doing a GET request to OneView"""
+    attempts = 3
+
+    for attempt_counter in range(attempts):
+        try:
+            connection = HTTPSConnection(
+                ov_config['ip'], context=ssl._create_unverified_context())
```

Comment:
Create a config option to allow the user to configure whether the toolkit will check invalid certificates or just ignored them.